pandas/core/arrays/sparse/accessor.py

```diff
@@ -339,12 +339,11 @@ def to_coo(self):
             dtype = dtype.subtype
 
         cols, rows, data = [], [], []
-        for col, name in enumerate(self._parent):
-            s = self._parent[name]
-            row = s.array.sp_index.to_int_index().indices
+        for col, (_, ser) in enumerate(self._parent.iteritems()):
```

Comment:
can you use itertuples here? can you run the asv's to see if any perf implications

Reply:
This is iterating over columns, so don't think itertuples would work here. Yep, will run asvs later